orttraining/orttraining/test/gradient/gradient_ops_test.cc

```diff
@@ -2423,6 +2423,54 @@ TEST(GradientCheckerTest, ClipGrad) {
   }
 }
 
+void GradientCheckerMinMaxGradHelper(std::string op) {
+  float max_error;
+  GradientChecker<float, float, float> gradient_checker;
+  OpDef op_def{op, kOnnxDomain, 11};
+
+  {
```

Comment:
need one case for matching values

Reply:
Does "matching values" here mean two inputs have the same value? If so, the third test case has some specified equal values. - Is it enough for this purpose?

Reply:
yes

Reply:
Test case 3 was meant to have some equal values for the two inputs, but I made a mistake here -- did not pass the x_datas.

When correcting the mistake, I realized that Max/Min is not differentiable at the equal value, and the test for equal case will just fail because the calculated numeric Jacobian and theoretical Jacobian are different. I discussed with @iK1D and think that maybe we can replace the "wrong" numeric Jacobian with the expected Jacobian (we know it for a specific input), and compare it with the theoretical one. Is this ok? 

Or should we just avoid the equal case? I see that Relu is similar, and ReluGrad's test does not have case for 0, where it's not differentiable.

Reply:
ok... since we know why the UT is failing, let's just skip the equal case for now. 
please add a comment to document this UT case though...